Formula/aptos.rb

```diff
@@ -0,0 +1,43 @@
+class Aptos < Formula
+  desc "Universal and fair access to decentralized assets for billions of people"
+  homepage "https://aptoslabs.com/"
+  url "https://github.com/aptos-labs/aptos-core/archive/refs/tags/aptos-cli-v1.0.3.tar.gz"
+  sha256 "670bb6cb841cb8a65294878af9a4f03d4cba2a598ab4550061fed3a4b1fe4e98"
+  license "Apache-2.0"
+
+  # This livecheck scans the releases folder and looks for all releases
+  # with matching regex of href="<URL>/tag/aptos-cli-v<SEMVER>". This
+  # is done to automatically check for new release versions of the CLI.
+  livecheck do
+    url "https://github.com/aptos-labs/aptos-core/releases"
+    strategy :page_match
+    regex(%r{href=["']?[^"' >]*?/tag/aptos-cli-v?(\d+(?:\.\d+)+)}i)
+  end
```

Comment:
this should be good enough, we have lots of examples in this codebase.
```suggestion
  livecheck do
    url :stable
    regex(/^aptos-cli[._-]v?(\d+(?:\.\d+)+)$/i)
  end
```

Reply:
Awesome thank you for the suggestion, @chenrui333, will add this in!